QRCoder/Base64QRCode.cs

```diff
@@ -1,33 +1,23 @@
-﻿#if NETFRAMEWORK || NETSTANDARD2_0 || NET5_0 || NET6_0_WINDOWS
+﻿#if !NETSTANDARD1_3
 using System;
 using System.Drawing;
 using System.Drawing.Imaging;
 using System.IO;
+using System.Runtime.InteropServices;
 using static QRCoder.Base64QRCode;
 using static QRCoder.QRCodeGenerator;
 
 namespace QRCoder
 {
-#if NET6_0_WINDOWS
-    [System.Runtime.Versioning.SupportedOSPlatform("windows")]
-#endif
     public class Base64QRCode : AbstractQRCode, IDisposable
     {
-        private QRCode qr;
-
         /// <summary>
         /// Constructor without params to be used in COM Objects connections
         /// </summary>
         public Base64QRCode() {
-            qr = new QRCode();
         }
 
         public Base64QRCode(QRCodeData data) : base(data) {
-            qr = new QRCode(data);
-        }
-
-        public override void SetQRCodeData(QRCodeData data) {
```

Comment:
Yes.  This method was overridden from the base class; I removed the override.  The method still exists (from the base class) and functions correctly.

Reply:
I can explain further if desired, but in short, it was overridden because the data was stored in an owned object rather than this object.  The class design is simplified so this is not the case anymore.